README.md

```diff
@@ -1 +1,3 @@
-# Animal-Competition-DGRM
\ No newline at end of file
+# Animal-Competition-DGRM
+
+We will have a large amount of animals (around 50) which will be categorized by features and size (little mammals, big birds, little amphibians) each class will have a different number for their Intelligence, Power, Defense,  Mobility, Health, Stealth, Stamina,  Intimidation, and Resolve. Players will be given a list of the stats above and get to choose their favourite this will raise the stat they choose and lower a different one. (The player could take a guess) Then the program will add up the stats and display the winner. (add environments) 
```

Comment:
It sounds like the results will be prett predictable if you play the game a few times because the animal strengths and weaknesses are pretty static.  Will you have environments that affect the strengths and weaknesses differently?  That would add a dimension of complexity for the player to solve.